tools/tertiary-analysis/scanpy/scanpy-plot-trajectory.xml

```diff
@@ -12,6 +12,7 @@ PYTHONIOENCODING=utf-8 scanpy-cli plot paga
     --use-key '${use_key}'
 #end if
     --layout ${layout}
+    --layout-loc ${layout-loc}
```

Comment:
```suggestion
    --layout-loc ${layout_loc}
```

Reply:
I suspect that this is the correct name @nomadscientist, layout-loc doesn't exists, and legend_loc matches with params. Also, cheetah variables cannot have `-` in the middle of the name.

Reply:
I have checked in scanpy scripts, so I will add this.